walk-in-interview/src/main/java/com/google/job/data/JobQuery.java

```diff
@@ -0,0 +1,100 @@
+package com.google.job.data;
+
+/** Class for the job listings query using builder pattern. */
+public final class JobQuery {
```

Comment:
This class look like a builder, it's supposed to build some immutable object, are we missing that part?

Reply:
I think we're just using the builder pattern here rather than an explicit builder.